### ISSUE TITLE
Bug in AB4: UI components hang while HelpWindow is active

### ISSUE THREAD

**Issue by erik0704** [nus-cs2103-AY1718S1/forum#181]
The bug exists in the current source code of the addressbook-level4 repo thus should exist on all team repos forked from it.

Reproduce steps:

- Start the app

- Run `help` command via the command box. The user guide helpwindow will pop out

- Switch back to the main app. Run any command. Every UI component will hang **except `PersonListPanel`**. The `ResultDisplay` box will not display anything. `select` command fails to load google search. All functionalities/commands which affect the model will(or seem to) work during this time, but the hanged UI components will only resume after the `HelpWindow` is closed.

I discovered this when during acceptance testing since need to see the user guide while operating the app.

This bug is quite interesting as it only happens when the HelpWindow is access via the command. If it is accessed via pressing F1 or manually navigate via the top bar then the bug does not happen.

**Comment by damithc** [comment]
Good catch.
@Zhiyuan-Amos can confirm?

**Comment by Zhiyuan-Amos** [comment]
Yup confirmed. Other students have already raised the issue in our repo already as well; I'll just make a link to this issue.

**Comment by damithc** [comment]
Thanks for raising this bug @erik0704 
Upstream repo is not doing any active development during this time, so we'll have to leave this bug to be fixed by one of you guys. Whoever fixes it can share with others. Alternatively, you can simply report it as a _known bug_ and of course you will not be penalized if the bug remains in your v1.5.
